sonarqube.yaml

```diff
@@ -0,0 +1,136 @@
+package:
+  name: sonarqube
+  version: "25.6.0.109173"
+  epoch: 0
+  description: SonarQube is an open source platform for continuous inspection of code quality (Commuunity Build)
```

Comment:
typo, should be community

Reply:
based on the recent decision, we will only have one sonarqube. And I think we should not name it as sonarqube-community